### ISSUE TITLE
comment-whitespace-inside work with `sass`(`scss`) and less comment

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#1049]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Throw error if does not whitespaces inside `sass` and `less` comments.

> Which rule, if any, is this issue related to?

e.g. `comment-whitespace-inside`

> What CSS is needed to reproduce this issue?

e.g.

```scss
//Test

.foo {
  color: pink;
}
```

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "rules": {
    "comment-whitespace-inside": "always"
  }
}
```

> Which version of stylelint are you using?

e.g. `5.3.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

e.g. "PostCSS plugin"

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

SCSS

> What did you expect to happen?

e.g. "No warnings to be flagged."

> What actually happened (e.g. what warnings or errors you are getting)?

Throw errors.

**Comment by ntwb** [comment]
Using `"comment-whitespace-inside": "always"` with `//Test` then you should see an error
Using `"comment-whitespace-inside": "always"` with `// Test` then you should **not** see an error

@evilebottnawi Did you write the correct CSS and configuration above? 

At the moment from your example code I'd say it was working as expected?

Existing SCSS tests: [/src/rules/comment-whitespace-inside/__tests__/index.js#L154](https://github.com/stylelint/stylelint/blob/master/src/rules/comment-whitespace-inside/__tests__/index.js#L154)

**Comment by evilebottnawi** [comment]
@ntwb i say about `//comment`, so that they are not ignored. Test show what now we ignored this type comments

**Comment by ntwb** [comment]
Right, so currently the tests only ignore SCSS & Less double slash comments, instead they should be included and enforced by the  existing  `comment-whitespace-inside` rule 👍

**Comment by jeddy3** [comment]
I believe our approach is to write rules for standard CSS *that don't fall over when they encounter common non-standard constructs*. We generally do this by ignoring the the non-standard constructs e.g. all the times we've ignored `@vars` and `$vars`.

Non-standard constructs are then tested for in plugins e.g. `stylelint-dollar-variable-pattern` etc.

So, I think this rule should stay as it is and you're more than welcome to write a plugin for your proposal. Perhaps called `comment-double-slash-space-after: "always"|"never"`. 

As you can see, this plugin can be specific to this use case i.e. neither `*-whitespace` (can't contains `/n`s) nor `*-inside` (only interested in what comes after the `//`) parts of the original rule make much sense in this context. You can also display a specific message e.g. "Expected a single space after "//" (comment-double-slash-space-after)".

Thoughts?

**Comment by evilebottnawi** [comment]
@jeddy3 agree, best solution - plugin
